doc/source/development/contributing.rst

```diff
@@ -135,9 +135,35 @@ operations. To install pandas from source, you need to compile these C
 extensions, which means you need a C compiler. This process depends on which
 platform you're using.
 
-* Windows: https://devguide.python.org/setup/#windows-compiling
-* Mac: https://devguide.python.org/setup/#macos
```

Comment:
For Mac, the https://devguide.python.org/setup/#macos link actually has some explanation about installing XCode, so maybe for this one we can keep the link until we have better instructions.

Reply:
I have replaced my Mac instructions with just the link above.

I've also changed formatting - happy to change back as well @jorisvandenbossche

Reply:
Formatting looks good!